scripts/apps/authoring/metadata/metadata.ts

```diff
@@ -1189,6 +1190,8 @@ export function MetadataService(api, subscribersService, vocabularies, $rootScop
                 _.each(items, (item) => {
                     self.values.customSubscribers.push({_id: item._id, name: item.name});
                 });
+
+                window.dispatchEvent(new CustomEvent('metadata-loaded'));
```

Comment:
Instead of dispatching an event here, call `fetchSubscribers` on `componentDidMount`. Also improve `fetchSubscribers` so it's cached - something like `fetchCities`.

Reply:
But it's not cities, it's subscribers

Reply:
I know it's not cities. I was suggesting to look at how caching is implemented in `fetchCities`

Reply:
Ok just re-read it yeah.